src/include/ceph_assert.h

```diff
@@ -120,7 +124,7 @@ using namespace ceph;
    {__STRING(expr), __FILE__, __LINE__, __CEPH_ASSERT_FUNCTION}; \
    ((expr) \
```

Comment:
Huh, where is the `likely` hinting? From the snippets it looks the compiler did a good job on guessing what's likely, what's not but I thought `ceph_assert` is hinted already. Perhaps we should add it.